README.md

```diff
@@ -124,49 +126,47 @@ We recommend the latest Rails 5.0 release.
 
-Generate a new Rails application using the template.
+NOTE: nodejs is preinstalled on most Mac computers and doesn't require a gem.  To test if nodejs is already installed, execute `node -v` in the terminal and the version of nodejs will be displayed if it is installed.
 
-```
-rails new my_app -m https://raw.githubusercontent.com/projecthydra-labs/hyrax/master/template.rb
-```
 
-Generating a new Rails application using Hyrax's template above takes cares of a number of steps for you, including:
+## Environments
 
-* Adding Hyrax (and any of its dependencies) to your application `Gemfile`, to declare that Hyrax is a dependency of your application
-* Running `bundle install`, to install Hyrax and its dependencies
-* Running Hyrax's install generator, to add a number of files that Hyrax requires within your Rails app, including e.g. database migrations
-* Loading all of Hyrax's database migrations into your application's database
-* Loading Hyrax's default workflows into your application's database
+Note here that the following commands assume you're setting up Hyrax in a development environment (using the Rails built-in development environment). If you're setting up a production or production-like environment, you may wish to tell Rails that by prepending `RAILS_ENV=production` to the commands that follow, e.g., `rails`, `rake`, `bundle`, and so on.
 
-## Generate a work type
 
-Hyrax allows you to specify your work types by using a generator. You may generate one or more of these work types.
+# Quick Start for a new Hyrax app
 
-Pass a (CamelCased) model name to Hyrax's work generator to get started, e.g.:
+The following steps need to be done in order to create a new Hyrax based app.  This is a quick list with links to the details.
 
-```
-rails generate hyrax:work Work
-```
+1. [Install prerequisite software](#install-prerequisite-software) - Follow all instructions carefully.
+1. [Create the Hyrax\-based app](#creating-a-hyrax-based-app) with `rails new my_app -m https://raw.githubusercontent.com/projecthydra-labs/hyrax/master/template.rb` (Replace my_app with the name of your app.)
+1. If using rubyracer for java runtime, uncomment in Gemfile and bundle install.  (Not needed if using nodejs.) ([more info](#javascript-runtime))
+1. [Start servers](#start-servers) with `rake hydra:server`  (e.g. solr, fedora, rails) - Stop with <CTRL><C>
+1. [Start background workers](#start-background-workers) (message queue) - several options for message queue
+1. [Create default administrative set](#create-default-administrative-set) with `rake hyrax:default_admin_set:create`
+1. [Load workflows](#load-workflows) with `rake hyrax:workflow:load`
+1. [Generate a work type](#generate-a-work-type) with `rails generate hyrax:work Work` (Replace Work with the name of your work type.)
```

Comment:
Also, this set of instructions duplicates a bunch of the instructions [below](https://github.com/projecthydra-labs/hyrax/pull/807/files#diff-04c6e90faac2675aa89e2176d2eec7d8R285) — maybe break out the initial part that's different and then have a single set of instructions for the common part so it's not duplicated?